{{cookiecutter.github_repo_name}}/Dockerfile

```diff
@@ -34,8 +39,8 @@ RUN git clone --depth=1 -b {{cookiecutter.odoo_version}} https://{{cookiecutter.
 ADD bin/ $ROCCO_PATH/bin
 RUN python3 $ROCCO_PATH/bin/submodules_processor.py $PROJECT_ADDONS_PATH {{cookiecutter.github_token}}
 
-# Install Addons Repo Requirements
-RUN if [ -f "$PROJECT_ADDONS_PATH/requirements.txt" ]; then python3 -m pip install -r $PROJECT_ADDONS_PATH/requirements.txt; fi
+# Install Addons Repo Requirements, this will drill down and find all requirements files
+RUN find "$PROJECT_ADDONS_PATH" -name "requirements.txt" -exec pip3 install -r {} \;
```

Comment:
Kudos per averti preso cura di questa cosa (e in modo migliore di come avrei fatto io che contavo di ciclare i submodule uno per uno)